org.jacoco.doc/docroot/doc/conventions.html

```diff
@@ -26,8 +26,14 @@
 <h2>Consistent Source Tree</h2>
 
 <p>
-  Update your sources and run the build locally before every check-in. Commit
-  only if the build succeeds.  
+  Follow the existing coding style and formatting rules. The projects contain
+  auto-formatting rules for the Eclipse IDE.
+</p>
+
+<p>
+  Update your local branches and run the build locally before every push. Push
+  only if the build succeeds and the configured compiler settings do not show
+  any warnings in the Eclipse IDE.
```

Comment:
@marchof This is quite strict requirement. And this rule is violated by me actually - most of the time I use IntelliJ IDEA ;) IMO should be allowed to contribute even without usage of Eclipse IDE. For example in case of IntelliJ IDEA it is possible to configure usage of Eclipse Code Formatter. Maybe even possible to enforce same warnings, but not sure.